modules/lvms-about-lvmcluster-cr.adoc

```diff
@@ -130,6 +130,24 @@ For example, if this field is set to 10, you can provision up to 10 times the am
 
 To disable over-provisioning, set this field to 1.
 
+|`thinPoolConfig.chunkSize`
+|`integer`
+|Specifies the statically calculated chunk size for the thin pool. It is used only when the `ChunkSizeCalculationPolicy` field is set to `Static`. The value for this field must be configured in the range 64 KiB to 1 GiB due to the underlying limitations of `lvm2`. 
+
+If this field is not configured, and the `ChunkSizeCalculationPolicy` field is set to `Static`, the default chunk size is `128KiB`.
+
+For more information, see "Overview of chunk size".
+
+|`thinPoolConfig.chunkSizeCalculationPolicy`
+|`string`
+|Specifies the policy to calculate the chunk size for the underlying volume group. You can set this field to either `Static` or `Host`. By default, this field is set to `Static`. 
+
+If this field is set to `Static`, the chunk size is set to the value of the `chunkSize` field. If the `chunkSize` field is not configured, chunk size is set to 128 KiB.
```

Comment:
```suggestion
If this field is set to `Static`, the chunk size is set to the value of the `chunkSize` field. If the `chunkSize` field is not configured, chunk size is set to `128 KiB`.
```

Previous example states `128KiB`, so does the value need the units and is there a space between the value and the unit?